.github/workflows/comment-to-trigger-ost.yaml

```diff
@@ -0,0 +1,19 @@
+name: comment '/ost' to trigger OST on a PR
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  trigger-ost:
+    if: |
+      github.event.issue.pull_request &&
+      startsWith(github.event.comment.body, '/ost') &&
+      (
+        github.event.comment.author_association == 'OWNER' ||
```

Comment:
no need for owner, please use/see https://github.com/oVirt/ovirt-system-tests/blob/master/.github/example-trigger.yaml

Reply:
also keep in mind that these checks work only for public ovirt members...so if something doesn't work please check their status first.
if you figure out a better check then please let me know:)

Reply:
I put OWNER in there so I could start the action in my personal fork.  Similar to the "/build rpms" action on ovirt-engine-nodejs-modules.  I wrote and tested it in my fork, but since I'm not a MEMBER or COLLABORATOR of a repo I own, had to add OWNER.  :man_shrugging:

Reply:
yes, but OST shouldn't start due to the runner being configured only for (some) oVirt repos. Please let me know if it does start!:)

Reply:
> yes, but OST shouldn't start due to the runner being configured only for (some) oVirt repos. Please let me know if it does start!:)

In my fork, the action starts but just sits there wait for a runner to become available ... which never happens.  But I could test that the comment does fire up the action.  I'll take the OWNER off, no need for it and it can still be consistent with the example.